Source/Shaders/BillboardCollectionVS.glsl

```diff
@@ -186,38 +224,19 @@ void main()
     float pixelOffsetScale = getNearFarScalar(pixelOffsetScaleByDistance, lengthSq);
     pixelOffset *= pixelOffsetScale;
 #endif
-
-    vec4 positionWC = czm_eyeToWindowCoordinates(positionEC);
-    
-    vec2 halfSize = imageSize * scale * czm_resolutionScale;
-    halfSize *= ((direction * 2.0) - 1.0);
-    
-    positionWC.xy += (origin * abs(halfSize));
     
-#if defined(ROTATION) || defined(ALIGNED_AXIS)
-    if (!all(equal(alignedAxis, vec3(0.0))) || rotation != 0.0)
+#ifdef TEST_GLOBE_DEPTH
+    vec4 offsetPosition = positionEC + vec4(0.0, 0.0, -positionEC.z * 0.05, 0.0);
+    vec4 wc = computePositionWindowCoordinates(offsetPosition, vec2(0.0, 0.0), scale, direction, origin, vec2(0.0), pixelOffset, alignedAxis, rotation);
+    float d = texture2D(czm_globeDepthTexture, wc.xy / czm_viewport.zw).r;
```

Comment:
OK, thanks for reminding me. I wasn't sure whether to throw or not. I also need to check if we can create a fbo with a floating-point color attachment. I've had issues before where floating-point textures were supported but the fbos were incomplete.